src/components/workshops/Carousel.js

```diff
@@ -0,0 +1,258 @@
+import React, { Component, Fragment } from 'react'
+import Slider from 'react-slick'
+
+import api from 'api'
+import styled from 'styled-components'
+import {
+  Box,
+  Card,
+  Container,
+  Flex,
+  Label,
+  Heading,
+  Button,
+  theme,
+} from '@hackclub/design-system'
+
+import CarouselProject from 'components/workshops/CarouselProject'
+import CarouselSubmissionForm from 'components/workshops/CarouselSubmissionForm'
+
+import 'slick-carousel/slick/slick.css'
+import 'slick-carousel/slick/slick-theme.css'
+
+const OriginalWrapper = styled(Flex).attrs({ mr: 1 })`
+  width: 200px;
+  flex-grow: 1;
+  flex-shrink: 1;
+  align-self: center;
+  overflow: hidden;
+  position: relative;
+
+  ${theme.mediaQueries.sm} {
+    width: 240px;
+  }
+  ${theme.mediaQueries.md} {
+    width: 350px;
+  }
+`
+
+const RehackWrapper = styled(Flex).attrs({ ml: 1 })`
+  width: 200px;
+  flex-direction: column;
+  flex-shrink: 1;
+  align-self: stretch;
+  overflow: hidden;
+  position: relative;
+
+  ${theme.mediaQueries.sm} {
+    width: 240px;
+  }
+  ${theme.mediaQueries.md} {
+    width: 350px;
+  }
+`
+
+const RehackSlider = styled(Slider)`
+  margin: 0;
+  overflow: hidden;
+  position: absolute;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+`
+
+const CarouselOuter = styled(Flex).attrs({
+  bg: theme.colors.smoke,
+  m: 0,
+  p: [2, 3, 4],
+  flexDirection: 'column',
+  align: 'center',
+})``
+
+const CarouselInner = styled(Flex).attrs({
+  mb: [2, 3, 4],
+})`
+  flex-direction: column;
+`
+class Carousel extends Component {
+  state = {
+    autoplay: false,
+    submitting: false,
+    submissionData: {
+      liveUrl: '',
+      codeUrl: '',
+    },
+    authed: false,
+    authData: {},
+    liveFrameStatus: 'empty',
+    liveFrameImage: null,
+  }
+  emptyProject = {
+    user: { username: '' },
+    empty: true,
+    live_url: '',
+    code_url: '',
+    screenshot: {
+      id: -1,
+      created_at: '',
+      updated_at: '',
+      type: '',
+      file_path: '',
+    },
+  }
+
+  constructor(props) {
+    super(props)
+
+    const { slug } = props
+
+    this.state.original = this.emptyProject
+    this.state.projects = []
+  }
+
+  componentDidMount() {
+    const self = this
+    const { slug } = this.props
+
+    api.get(`v1/workshops/${slug}/projects`).then(projects => {
```

Comment:
Can delete the lines above &
```suggestion
    api.get(`v1/workshops/${this.props.slug}/projects`).then(projects => {
```